src/classworks/classworks.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Controller, Post, UseInterceptors, UploadedFile, Param, Body } from "@nestjs/common";
+import { FileInterceptor } from "@nestjs/platform-express";
+
+import { ClassworksService } from "./classworks.service";
+
+@Controller("classworks")
+export class ClassworksController {
+  constructor(private readonly classworksService: ClassworksService) {}
+
```

Comment:
Where are you planning to put other crud endpoints for Exam and Assignments?

Reply:
Since you have seperate models for exam, assignment and resources. It's best to have seperate controllers for them. The naming should follow the same name as model.

For your case here we should Name the folder `resources`, and the controller, module and service should use the same prefix.